static/app/gettingStartedDocs/apple/apple-macos.tsx

```diff
@@ -195,8 +199,91 @@ const onboarding: OnboardingConfig = {
   ],
 };
 
+export const appleFeedbackOnboarding: OnboardingConfig = {
+  introduction: () => getCrashReportApiIntroduction(),
+  install: () => [
+    {
+      type: StepType.INSTALL,
+      description: getCrashReportInstallDescription(),
+      configurations: [
+        {
+          code: [
+            {
+              label: 'Swift',
+              value: 'swift',
+              language: 'swift',
+              code: `import Sentry
+
+let eventId = SentrySDK.capture(message: "My message.")
+
+let userFeedback = UserFeedback(eventId: eventId)
+userFeedback.comments = "It broke."
+userFeedback.email = "john.doe@example.com"
+userFeedback.name = "John Doe"
+SentrySDK.capture(userFeedback: userFeedback)`,
+            },
+            {
+              label: 'Objective-C',
+              value: 'c',
+              language: 'c',
+              code: `@import Sentry;
+
+SentryId *eventId = [SentrySDK captureMessage:@"My message"];
+
+SentryUserFeedback *userFeedback = [[SentryUserFeedback alloc] initWithEventId:eventId];
+userFeedback.comments = @"It broke.";
+userFeedback.email = @"john.doe@example.com";
+userFeedback.name = @"John Doe";
+[SentrySDK captureUserFeedback:userFeedback];`,
+            },
+          ],
+        },
+        {
+          description: tct(
+            'To capture user feedback regarding a crash, use the [code:SentryOptions.onCrashedLastRun] callback. This callback gets called shortly after the initialization of the SDK when the last program execution terminated with a crash. It is not guaranteed that this is called on the main thread.',
+            {code: <code />}
+          ),
+          code: [
+            {
+              label: 'Swift',
+              value: 'swift',
+              language: 'swift',
+              code: `import Sentry
+
+SentrySDK.start { options in
+    options.dsn = "https://16f104a1b76e4d69a1d7d075312254a7@o19635.ingest.sentry.io/5471241"
```

Comment:
I think you want to inject the dsn from the params 😉

Reply:
good catch!!